src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -31,15 +31,19 @@ public class AnalysisService : IDisposable
         /// Defines the list of Script Analyzer rules to include by default if
         /// no settings file is specified.
         /// </summary>
-        private static readonly string[] IncludedRules = {
+        private static readonly string[] IncludedRules = new string[]
+        {
+            "PSUseToExportFieldsInManifest",
+            "PSMisleadingBacktick",
+            "PSAvoidUsingCmdletAliases",
             "PSUseApprovedVerbs",
+            "PSAvoidUsingPlainTextForPassword",
             "PSReservedCmdletChar",
             "PSReservedParams",
             "PSShouldProcess",
             "PSMissingModuleManifestField",
             "PSAvoidDefaultValueSwitchParameter",
-            "PSUseDeclaredVarsMoreThanAssigments",
-            "PSMisleadingBacktick",
+            "PSUseDeclaredVarsMoreThanAssigments"
```

Comment:
We should test the new rule additions.  We got a lot of blow-back from the community early on because of lots of green squiggles in folks' scripts.  Also, we should add a note in the comments for this section, that any updates here should be also made in the https://github.com/PowerShell/vscode-powershell/blob/master/examples/PSScriptAnalyzerSettings.psd1 file as well.  

BTW it is more palatable for folks when the green squiggles underline just the bare minimum of the script as opposed to say entire functions.  :-)   BTW that also helps with not having "overlapping" script extents for multiple rule violations.